components/application-registry/scripts/can-i-commit.sh

```diff
@@ -24,15 +24,15 @@ if [ $? != 0 ]
 fi
 
 ##
-# MAKE BUILD
+# MAKE VERIFY
```

Comment:
Do we still need this file? I guess we can do basically the same just by running `make verify`.

Reply:
Done!